<ui version="4.0" >
 <class>EditUsersListDialogClass</class>
 <widget class="QDialog" name="EditUsersListDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>382</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>5</number>
   </property>
   <item>
    <widget class="QTableWidget" name="tbwTable" >
     <property name="editTriggers" >
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
     <property name="columnCount" >
      <number>2</number>
     </property>
     <column>
      <property name="text" >
       <string>Jid</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Reason</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="pbtAdd" >
       <property name="text" >
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtDelete" >
       <property name="text" >
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="btbButtons" >
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>tbwTable</tabstop>
  <tabstop>pbtAdd</tabstop>
  <tabstop>pbtDelete</tabstop>
  <tabstop>btbButtons</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btbButtons</sender>
   <signal>accepted()</signal>
   <receiver>EditUsersListDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>250</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel" >
     <x>225</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btbButtons</sender>
   <signal>rejected()</signal>
   <receiver>EditUsersListDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>359</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel" >
     <x>388</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
